controllers/storageoscluster_controller.go

```diff
@@ -52,7 +52,7 @@ func NewStorageOSClusterReconciler(mgr ctrl.Manager) *StorageOSClusterReconciler
 // +kubebuilder:rbac:groups=apps,resources=statefulsets;daemonsets;deployments;replicasets,verbs=*
 // +kubebuilder:rbac:groups="",resources=nodes,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups="",resources=pods,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups="",resources=events;namespaces;serviceaccounts;secrets;services;services/status;services/finalizers;persistentvolumeclaims;persistentvolumeclaims/status;persistentvolumes;configmaps;configmaps/status;replicationcontrollers;pods/binding;pods/status;endpoints;endpoints/status,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups="",resources=events;namespaces;serviceaccounts;secrets;services;services/status;services/finalizers;persistentvolumeclaims;persistentvolumeclaims/status;persistentvolumes;configmaps;configmaps/status;replicationcontrollers;pods/binding;pods/status;endpoints;endpoints/status;resourcequotas,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Do we need all the permissions for RQ?

Reply:
I guess not, but we do need create, update and delete 🤷‍♂️